templates/systemd.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Mattermost
+After=syslog.target network.target postgresql-9.6.service
```

Comment:
please make this version configurable.
I bet there is a fitting variable in our postgres role which you can use here.

Reply:
done

Reply:
and there is not